src/blocks/Questions/models.ts

```diff
@@ -1,5 +1,6 @@
 export const FaqMicrodataValues = {
     PageType: 'https://schema.org/FAQPage',
+    PageId: '/faq',
```

Comment:
if we have multiple FAQ's, will everyone have the same one? Is this normal for robots?

Reply:
the point was to give different faq's links to the same page. google's rich results test does not seem to have a problem with it